lambda_function.py

```diff
@@ -18,7 +18,12 @@ def lambda_handler(event, context):
                 omeka_data[text['element']['name']] = text['text']
             if item['files']['count'] > 0:
                 files = requests.get(item['files']['url'])
-                media = [{'urls': f['file_urls']} for f in files.json()]
+                media = sorted(
+                    [{'urls': f['file_urls'], 'order': f['order']} for f in files.json()],
+                    key= lambda i: i['order'] if i['order'] else 100000
```

Comment:
Even though some items no explicit order value, I wonder if we can make this a little more deterministic and maybe consistent with how it works on the Omeka site itself, if we fall back to the file `id` when there is no explicit ordering. What do you think?

Reply:
Yeah, that makes total sense. Great idea.